bin/s2nc.c

```diff
@@ -285,7 +293,7 @@ int main(int argc, char *const *argv)
 
     while (1) {
         int option_index = 0;
-        int c = getopt_long(argc, argv, "a:c:ehn:m:sf:d:l:k:D:t:irTCBL:P:E:", long_options, &option_index);
+        int c = getopt_long(argc, argv, "a:c:ehn:m:sf:d:l:k:D:t:iro:I:TCBL:P:E:", long_options, &option_index);
```

Comment:
Not sure I understand how this works, when is a colon required between arguments?

Reply:
A colon in the optstring indicates that the character requires an argument. The two new characters I'm adding require that the user specify the file path where they want the session ticket to be written to/read from.